### ISSUE TITLE
Documentation site - Client libraries: possible content audit

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#173]
Things to review and consider

Are the images and grids accessible?

Is "clients" sufficiently descriptive in page title https://documentation.notification.canada.ca/en/clients.html?

It looks like the audience for this section is only developers. If so, could we indicate that near the start of the page? If not, can we revise this page for plain language? If not, can we define all the abbreviations?

Potential fix
We cannot design a potential fix until the team decides the audience for and purpose of this content.  Reconsider this page once team provides direction.

![image](https://user-images.githubusercontent.com/51717705/140793628-448fda04-4e14-49ed-91ad-cbd13e070a2f.png)

**Comment by amazingphilippe** [comment]
Page titles for the documentation site follow this pattern:

`Page title` | GC Notify

Instead of simply "GC Notify" we could instead use "GC Notify API documentation"
